ci/infra/testrunner/tests/test_addon_upgrade_apply.py

```diff
@@ -0,0 +1,21 @@
+import yaml
```

Comment:
I would not divide addons tests in separate suites, those tests are not disruptive, neither having different environment considerations, they can be all part of the same suite.